src/web/server/document.tsx

```diff
@@ -42,9 +42,16 @@ interface Props {
 	data: DCRServerDocumentData;
 }
 
+const makeTitle = (CAPI: CAPIType): string => {
+	if (CAPI.pillar === 'opinion' && CAPI.author.byline) {
+		return `${CAPI.headline} | ${CAPI.author.byline} | The Guardian`;
+	}
+	return `${CAPI.headline} | ${CAPI.sectionLabel} | The Guardian`;
+};
```

Comment:
We have a convention of `decideSomething` and rather than direction on CAPI.pillar, work with the Format type: 

```suggestion
const decideTitle = (CAPI: CAPIType): string => {
	if (format.theme === Pillar.Opinion && CAPI.author.byline) {
		return `${CAPI.headline} | ${CAPI.author.byline} | The Guardian`;
	}
	return `${CAPI.headline} | ${CAPI.sectionLabel} | The Guardian`;
};
```

Reply:
I also have a feeling you'll want to do this in the metadata too for social sharing?